packages/cojson/src/PeerKnownStates.ts

```diff
@@ -1,41 +1,51 @@
 import { RawCoID, SessionID } from "./ids.js";
 import {
+  cloneKnownState,
+  combineKnownStates,
   CoValueKnownState,
-  combinedKnownStates,
   emptyKnownState,
-} from "./sync.js";
+  updateSessionCounter,
+} from "./knownState.js";
 
 export class PeerKnownStates {
   private coValues = new Map<RawCoID, CoValueKnownState>();
 
+  getKnownState(id: RawCoID) {
+    const knownState = this.coValues.get(id);
+
+    if (!knownState) {
+      const knownState = emptyKnownState(id);
+      this.coValues.set(id, knownState);
+      return knownState;
+    }
+
+    return knownState;
+  }
```

Comment:
Created a known state getter to centralize the empty fallback and remove the `coValue.set` when not necessary.